api_tester/lib/models/customer_info_wrapper_api_test.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:purchases_flutter/object_wrappers.dart';
+
+// ignore_for_file: unused_element
+// ignore_for_file: unused_local_variable
+class _CustomerInfoApiTest {
+  void _checkFromJsonFactory(Map<String, dynamic> json) {
```

Comment:
I added `fromJson` and `toJson` checks to all these `freeze` models... I don't think that's meant to be public, but it currently is (don't think there is an easy fix for that though), so I thought I would add them as part of the API tests... Lmk if you prefer me to remove them.